pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -309,8 +309,8 @@ func NewCertRotationController(
 		certrotation.SigningRotation{
 			Namespace:     operatorclient.OperatorNamespace,
 			Name:          "kube-control-plane-signer",
-			Validity:      8 * time.Hour, // to be 10 days
-			Refresh:       4 * time.Hour, // to be 4 days
+			Validity:      30 * 24 * time.Hour, // 30 days to be in-sync with cluster-signing-duration set for kube-cluster-manager
+			Refresh:       15 * 24 * time.Hour, // 50% of validity time to be able to respond to changes
```

Comment:
what about all the other values? This does not help much, it's only changing the CA validity.

Reply:
> what about all the other values? This does not help much, it's only changing the CA validity.

yeah, you need to get the others